app/gateway/changelog.md

```diff
@@ -753,6 +753,50 @@ This should be a 400 because the configuration is invalid.
 * When the OpenID Connect (OIDC) plugin is configured to reference HashiCorp Vault in the `config.client_secret` field (for example, `{vault://hcv/clientSecret}`),
 it does not look up the secret correctly.
 
+## 3.2.2.4
+**Release Date** 2023/08/21
+
+### Enterprise
+
+#### Breaking Changes
+
+- Ubuntu 18.04 artifacts are no longer supported as they are end-of-life.
+- Amazon Linux 2022 artifacts are renamed to Amazon Linux 2023, based on AWS's own renaming.
+
+#### Deprecations
+
+- **CentOS packages are now removed from the release and are no longer supported in future versions.**
```

Comment:
Can you remove this entry? We deprecated CentOS packages back in 3.0. They accidentally built one CentOS package in some random patch release again, I don't remember which; but from the docs perspective, we haven't had CentOS in our docs since 2.8.x